### DISCUSSION TITLE
v0.2.2-beta.1

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#170] [General]
## What's New
- Always show Dashboard icon even if whatsapp integration is broken
- Notify and disable some features in dashboard when whatsapp integration is broken (will do in dashboard icon and tray in the future)
- Now WALC will open existing window when launching the app again

### Tip
Did you know that WALC has a hidden menu? Press alt to show it! The menu has useful shortcuts like the nifty chat shortcut. Productivity goes brrr... 📈📈

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.2.2-beta.1'>v0.2.2-beta.1</a>.</em>

**Comment by albakhtari** [comment]
And my `Ctrl+R` works! 
Thanks @Zzombiee2361!!

**Reply by daffamumtaz2361** [reply]
Wait what? I never added that. `Ctrl+R` isn't reloading on mine, is that whatsapp beta thing?

**Reply by daffamumtaz2361** [reply]
Also, worry not, cause lock screen is next! <sup>if I had the time that is</sup>

**Reply by albakhtari** [reply]
> Wait what? I never added that. `Ctrl+R` isn't reloading on mine, is that whatsapp beta thing?

I guess the broken integration is what fixed it.

**Comment by albakhtari** [comment]
Will the tray icon badge work with the beta now?

**Reply by albakhtari** [reply]
I confirm, the tray icon message badge thingy **IS WORKING WITH THE MULTI-DEVICE BETA!!**
WALC is now as perfect as it can get :sunglasses:

**Comment by albakhtari** [comment]
The tray icon message count doesn't work anymore after the latest update, neither do the desktop notifications on the multi-device beta (~I know @Zzombiee2361, you can't keep updating for a beta but just one more time, please :pleading_face:~)
EDIT: The problem persists without the beta
